libraries/ui/src/default-config/tailwind.css

```
@@ -113,6 +113,17 @@
     @apply w-full max-w-max-width mx-auto px-spacing-x pt-6 pb-10 border-b border-color-divider overflow-hidden flex flex-col;
 }
 
+@utility nav-link-animation {
+    @apply relative transition-[translate] duration-200;
+    @apply hover:text-current hover:-translate-y-[4px];
+    @apply after:content-[""] after:invisible after:absolute after:bg-bluedot-normal after:transition-[translate] after:duration-200 after:left-0 after:bottom-0 after:w-full after:h-[2px];
```

this looks great! i like how you broke it up into different statements. could we add a comment providing context for why we took this approach as well?
```suggestion
     /* underline is applied by after psuedoelement for consistency across text and incorporated SVGs */
    @apply after:content-[""] after:invisible after:absolute after:bg-bluedot-normal after:transition-[translate] after:duration-200 after:left-0 after:bottom-0 after:w-full after:h-[2px];
```